servicemesh-install/topics/appendix-crd-old.adoc

```diff
@@ -0,0 +1,283 @@
+[[creating-custom-resource-pre-tp10]]
```

Comment:
@geekspertise if the user decides to use the old resource definition, we need to make sure that that the user needs to apply the resource to `-n istio-operator` instead of `-n istio-system` otherwise it will fail.

Reply:
I added this note:

If you are using the previous version of the custom resource definition (linked), apply the resource to istio-operator instead of istio-system or the install will fail.